test/e2e/aws_test.go

```diff
@@ -57,7 +57,14 @@ var _ = Describe("functional tests", func() {
 		cpMachinePrefix         string
 		cpAWSMachinePrefix      string
 		cpBootstrapConfigPrefix string
+		mdBootstrapConfig 		string
```

Comment:
This looks like it could use running through `gofmt`

Reply:
Used go fmt